### PR TITLE
 set expectations with users about the purpose and intention of havin…

### PR DESCRIPTION
**Pull request by keshav-pivotal**
…g a Route CRD and route resource

> Thanks for contributing to cf-for-k8s!
>
> We've designed this PR template to speed up the PR review and merge process - please use it.

No

## Things to remember
- Make sure this PR is based off the `develop` branch
- Include any links to related PRs, issues, stories, slack discussions, etc... that will help establish context.
- Is there anything else of note that the reviewers should know about this change?
- Our CI uses concourse task files from HEAD of the `develop` branch. If your PR includes/requires CI changes, please ping the RelInt interrupt and they'll help apply the CI changes for you.
- Checkout the [contributing guidelines](https://github.com/cloudfoundry/cf-for-k8s/blob/develop/community/CONTRIBUTING.md)

### CODE REVIEW

File: docs/platform_operators/warning-regarding-route_crd-usage [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Currently, when a user creates a route the Cloud Controller creates a route resource as defined by [Route CRD](https://docs.google.com/document/d/1DF7eTBut1I74w_sVaQ4eeF74iQes1nG3iUv7iJ7E35U/edit#heading=h.1z6hvc8yon6k) writes it to the K8s API.
+The Route Controller reads the route resource from the Kubernetes API.
+Once the Route Controller reads the route resource it creates istio services and virtual services and writes them to the K8s API. Istiod then reads the virtual services and services from the K8s API.
+
+Operators are welcome to "peek into the system" and view the route resource. However, we treat it as an implementation detail and consider the Route CRD subject to change in the future. 
+We do not support the route resource for any uscases apart from the one described above.
```
Comment (reneighbor) [inline docs/platform_operators/warning-regarding-route_crd-usage:6]: typo in "uscases"

File: docs/platform_operators/warning-regarding-route_crd-usage [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Currently, when a user creates a route the Cloud Controller creates a route resource as defined by [Route CRD](https://docs.google.com/document/d/1DF7eTBut1I74w_sVaQ4eeF74iQes1nG3iUv7iJ7E35U/edit#heading=h.1z6hvc8yon6k) writes it to the K8s API.
```
Comment (reneighbor) [inline docs/platform_operators/warning-regarding-route_crd-usage:1]: Should be in markdown. Needs some formatting, ie heading seperate from text. See other docs in this directory for example.

**Review by reneighbor — COMMENTED** [review]
Out of curiosity, are we sure we need this PR? It seems like the major theme is covered in a page we already have: https://github.com/cloudfoundry/cf-for-k8s/blob/master/docs/platform_operators/warning-regarding-kubectl-usage.md

The kubectl warning is important to call out however. Maybe we should put it in a different spot, such as the project README? /cc @paulcwarren

**Comment by jamespollard8** [comment]
@keshav-pivotal I'm going to close this for now but please feel free to re-open it as you see fit

**Pull request closed without merging**